crates/nu-std/CONTRIBUTING.md

```diff
@@ -0,0 +1,191 @@
+(strongly) consider writing up a section for the release notes.  
+Currently, release notes are maintained in a different repo, [nushell.github.io](https://github.com/nushell/nushell.github.io). 
+Make your change in a local clone of that repo and submit a PR to the release notes repo to get it integrated.
+
+## Developing
+
+### Setup
+To work on the standard library, you won't necessarily need to compile or debug the interpreter itself, 
+so you might not need to install the Rust toolchain. 
+But if you want to do that, see [nushell CONTRIBUTING.md](https://github.com/nushell/nushell/blob/main/CONTRIBUTING.md).
+
+1. Clone the Nushell repo containing the standard library.  
+Currently, that's the [Nushell interpreter source repo](https://github.com/nushell/nushell).
+    ```
+    git clone https://github.com/nushell/nushell
+    ```
+
+2. Arrange a runnable Nushell interpreter  
+  Either install a [pre-built distribution](https://github.com/nushell/nushell/releases), or build from source:
+    ```
+    git clone https://github.com/nushell/nushell
+    cd nushell
+    cargo run
+    ```
+1. Focus your IDE on the standard library folder  
+(Currently, it is a Rust crate within the Nushell repo)
+
+    ```
+    cd /path/to/nushell/repo
+    code .  # opinionated: we love vscode!
+    (open folder crates/nu-std in IDE)
+   ```
+### The PR
+If your idea is mostly a bug fix, you might not open a PR till you've got a fix (nearly)ready to go.  
+
+But if you're considering new commands or substantial or (even simple) breaking changes, 
+it's good to advertise your proposed work to other collaborators for their awareness 
+and to solicit their early feedback. 
+Often, this feedback will result in a better and more maintainable solution long term, 
+so give your fellow collaborators the opportunity.  Someday, they'll return the favor...
+
+To open a PR before you've got code ready to go:
+  1. create a feature branch in your fork of the repo
+  2. make a trivial code change and push it to Github
+  4. open a PR on the standard library repo
+  5. **mark it as a work in progress (WIP)**  
+This signals that you are not yet asking to have the PR formally reviewed and merged.
+  6. provide a preliminary description of your proposed change.     
+Your description should include the external interface for your feature, 
+a draft of the command arguments and signature, 
+new/changed environment variables and any other user-visible changes.
+
+### Design considerations
+The standard library consists mostly of Nushell custom commands and their associated environment variables.  
+For background on these Nushell features, see 
+[Modules chapter of the Nushell book](https://www.nushell.sh/book/modules.html). 
+Existing code in the standard library provides pretty good examples to work from.  
+
+1. Create / update the source and test files for a custom command `foo` as follows:
+   ```shell
+   # source file 
+   /path/to/nushell/crates/nu-std/lib/foo.nu  
```

Comment:
just wondering if we should only write `crates/nu-std/...` instead of the generic `/path/to/...` ?

Reply:
I have to be consistent about paths, one way or the other....

Reply:
i might prefere `crates/nu-std/...` a bit
- removes a bit of noise from `/path/to/...`
- a bit shorter

BUT it assumes you are in the root of `nushell` (which is not that much of a strong assumption?)

:bulb: i do not have a strong preference here